### ISSUE TITLE
[REQUEST]

### ISSUE THREAD

**Issue by gabriellluz** [Hellowlol/HTPC-Manager#521]
I got many apps running on my old home server and the way HTPC Manager is organized, my apps list got messed up.

With HTPC manager, I can see all of the apps, but I suggest we could have an option to see our apps like a mosaic with all of them. It'be nice if we squares for every app. :)

**Comment by Hellowlol** [comment]
I'm not sure I follow. Are you referring to the menu at the top?

**Comment by gabriellluz** [comment]
Yes, exactly sir. That menus is not good if you have a lot of apps.
